Added optional parameter to specify a param name.

gbourassa opened the pull request.

meiikan commented:
Bon matin !


J'espère que tout va bien. Je vous envois ce mail parce que je suis encore dans votre répertoire, je crois que ce serait un bon move de me retirer [😉] .


Merci d'avance, bonne journée!


David Bilodeau



________________________________
De : Guillaume Bourassa <notifications@github.com>
Envoyé : 18 janvier 2017 06:59
À : mirego/gogradle
Cc : Subscribed
Objet : [mirego/gogradle] Added optional parameter to specify a param name. (#2)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/mirego/gogradle/pull/2

[https://avatars1.githubusercontent.com/u/2066229?v=3&s=400]<https://github.com/mirego/gogradle/pull/2>

Added optional parameter to specify a param name. by gbourassa · Pull Request #2 · mirego/gogradle<https://github.com/mirego/gogradle/pull/2>
github.com
gogradle - Contains some useful gradle scripts.


Commit Summary

  *   Added optional parameter to specify a param name.

File Changes

  *   M README.md<https://github.com/mirego/gogradle/pull/2/files#diff-0> (14)
  *   M increment.gradle<https://github.com/mirego/gogradle/pull/2/files#diff-1> (7)

Patch Links:

  *   https://github.com/mirego/gogradle/pull/2.patch
  *   https://github.com/mirego/gogradle/pull/2.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mirego/gogradle/pull/2>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ALYII1ymhs7FvWYY8Q__q76G9zFD10Udks5rTf6rgaJpZM4LmzSq>.

gbourassa commented:
C’est un repo public, et tu le watch. C'est legit. 
Si tu ne veux plus recevoir de courriels concernant ce repo, tu peux arrêter de le watcher.


Merci!

On January 18, 2017 at 11:00:27 AM, David Bilodeau (notifications@github.com)
wrote:

Bon matin !


J'espère que tout va bien. Je vous envois ce mail parce que je suis encore
dans votre répertoire, je crois que ce serait un bon move de me retirer
[😉] .


Merci d'avance, bonne journée!


David Bilodeau



________________________________
De : Guillaume Bourassa <notifications@github.com>
Envoyé : 18 janvier 2017 06:59
À : mirego/gogradle
Cc : Subscribed
Objet : [mirego/gogradle] Added optional parameter to specify a param name.
(#2)

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/mirego/gogradle/pull/2

[https://avatars1.githubusercontent.com/u/2066229?v=3&s=400]<
https://github.com/mirego/gogradle/pull/2>

Added optional parameter to specify a param name. by gbourassa · Pull
Request #2 · mirego/gogradle<https://github.com/mirego/gogradle/pull/2>
github.com
gogradle - Contains some useful gradle scripts.


Commit Summary

* Added optional parameter to specify a param name.

File Changes

* M README.md<https://github.com/mirego/gogradle/pull/2/files#diff-0> (14)
* M increment.gradle<https://github.com/mirego/gogradle/pull/2/files#diff-1>
(7)

Patch Links:

* https://github.com/mirego/gogradle/pull/2.patch
* https://github.com/mirego/gogradle/pull/2.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<
https://github.com/mirego/gogradle/pull/2>, or mute the thread<
https://github.com/notifications/unsubscribe-auth/ALYII1ymhs7FvWYY8Q__q76G9zFD10Udks5rTf6rgaJpZM4LmzSq
>.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/mirego/gogradle/pull/2#issuecomment-273516136>, or mute
the thread
<https://github.com/notifications/unsubscribe-auth/AB-HNQrMURx6gWLAqhb6NOyGpzJiQu34ks5rTjcagaJpZM4LmzSq>
.
